extensions/amp-story-panning-media/README.md

```diff
@@ -0,0 +1,156 @@
+
+<!---
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-panning-media example" width="304" height="548" layout="fixed" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif">
+    <noscript>
+    <img alt="amp-story-panning-media example" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+The `amp-story-panning-media` component provides a way to pan and zoom an image between pages in [Web Stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories).
+
+## Usage
+
+Use the `amp-story-panning-media` to transition an image between pages. The component can animate between position and zoom.
+
+## Environment setup
+
+Add the following import to your AMP documents:
+
+```html
+<script async custom-element="amp-story-panning-media" src="https://cdn.ampproject.org/v0/amp-story-panning-media-0.1.js"><script>
+```
+
+### Configuration
+
+The `amp-story-panning-media` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories). Components associate with each other when they have a child with the same `src`.
+
+```html
+<amp-story-page>
+  <amp-story-grid-layer>
+    <amp-story-panning-media layout="fill">
+      <amp-img layout="fill" src="assets/world-map.jpg" x="-10%" y="-20%" zoom=".8" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+
+<amp-story-page>
+  <amp-story-grid-layer>
+    <amp-story-panning-media layout="fill">
+      <amp-img layout="fill" src="assets/world-map.jpg" x="20%" y="30%" zoom="2" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+```
+
+## Attributes
+
+### x {string} (optional)
+
+Specifies the horizontal position in percentage.  
+Centered on 0 (default) with positive values moving the image to the right and negative to the left.
+
+### y {string} (optional)
+
+Specifies the vertical position in percentage.
+Centered on 0 (default) with positive values moving the image to the top and negative to the bottom.
+
+### zoom {float} (optional)
```

Comment:
Haven't seen anywhere in amp.dev that these titles can show the type they expect. @CrystalOnScript is this in line with the component documentation? Also, all attributes are strings because of HTML syntax, but if you want to explain that `x`, `y` and `zoom` are numbers / percentages, you can talk about the range here (eg: -50% to 50%) and what they mean (left align vs right align)

Reply:
Updated to remove the types and explain values in description.

This was following [amp-story-360 attributes](https://amp.dev/documentation/components/amp-story-360/?format=stories#attributes). I can update that in a separate PR.